README.md

```diff
@@ -11,6 +11,9 @@ I am happy to accept any [bug reports](https://github.com/mattdy/MMM-TeslaFi/iss
 ## Screenshot
 
 ![screenshot.png](doc/screenshot.png)
+![screenshot.png](doc/Driving-With-Map.png)
```

Comment:
Could this be improved by adding an caption saying what options are selected on each screenshot? Looks a little confusing otherwise!

Reply:
I've added which fields are active in each screenshot (updated one and removed a redundant screenshot)